assimilation_code/modules/assimilation/filter_mod.rst

```diff
@@ -0,0 +1,501 @@
++------------------------------+---------------------+-------------------------------------------+
+| compute_posterior            | logical             | If ``.false.``, skip computing posterior  |
+|                              |                     | forward operators and do not write        |
+|                              |                     | posterior values in the obs_seq.final     |
+|                              |                     | file. Saves time and memory. Posterior    |
+|                              |                     | inflation is not possible. For backwards  |
+|                              |                     | compatibility the default ``.true.``      |
++------------------------------+---------------------+-------------------------------------------+
+| ens_size                     | integer             | Size of ensemble.                         |
++------------------------------+---------------------+-------------------------------------------+
+| num_groups                   | integer             | Number of groups for hierarchical filter. |
+|                              |                     | It should evenly divide ens_size.         |
++------------------------------+---------------------+-------------------------------------------+
+| distributed_state            | logical             | ``.true.`` means the ensemble data is     |
+|                              |                     | distributed across all tasks as it is     |
+|                              |                     | read in, so a single task never has to    |
+|                              |                     | have enough memory to store the data for  |
+|                              |                     | an ensemble member. Large models should   |
+|                              |                     | always set this to ``.true.``, while for  |
+|                              |                     | small models it may be faster to set this |
+|                              |                     | to ``.false.`` This is different from     |
+|                              |                     | ``&assim_tools_mod :: distributed_mean``. |
++------------------------------+---------------------+-------------------------------------------+
+| async                        | integer             | Controls method for advancing model:      |
+|                              |                     | | 0 is subroutine call                    |
+|                              |                     | | 2 is shell command                      |
+|                              |                     | | 4 is mpi-job script                     |
+|                              |                     | | Ignored if filter is not controlling    |
+|                              |                     | the model advance, e.g. in CESM, WRF, etc |
++------------------------------+---------------------+-------------------------------------------+
```

Comment:
This cell is rendering with | in the text rather than line breaks. 
![Screen Shot 2021-04-22 at 10 00 06 AM](https://user-images.githubusercontent.com/20047007/115746452-9573ba00-a351-11eb-87a9-0194c5b02823.png)

Reply:
**I know!** And if you can figure out how to make those items appear on separate lines - I'll owe you one.  I tried several methods to no avail.   In the '914-character-wide' table, this syntax was used to create line breaks.  Why it's not working with the new table is quite frustrating.

Reply:
I've "fixed" that by making it an itemized list.
I'm wondering whether the '|' behaves differently in the first columns vs. the others.
I'm almost done with the whole file, and was planning to commit it,
but there's an SVT fire drill, then a meeting at 11, so it might be after noon before I get to it.